x-pack/test/alerting_api_integration/spaces_only/tests/alerting/ephemeral.ts

```diff
@@ -0,0 +1,129 @@
+
+  const esTestIndexTool = new ESTestIndexTool(es, retry);
+
+  describe('ephemeral', () => {
+    const objectRemover = new ObjectRemover(supertest);
+
+    beforeEach(async () => {
+      await esTestIndexTool.destroy();
+      await esTestIndexTool.setup();
+    });
+    afterEach(async () => await esTestIndexTool.destroy());
+    after(async () => {
+      await objectRemover.removeAll();
+    });
+
+    it('should execute all requests, when some will be ephemeral and some not', async () => {
+      const nonEphemeralTasks = 3;
+      const actionPromises = [];
+      for (let i = 0; i < DEFAULT_MAX_EPHEMERAL_ACTIONS_PER_ALERT + nonEphemeralTasks; i++) {
+        actionPromises.push(
+          supertest
+            .post(`${getUrlPrefix(Spaces.space1.id)}/api/actions/connector`)
+            .set('kbn-xsrf', 'foo')
+            .send({
+              name: `My action${i}`,
+              connector_type_id: 'test.index-record',
+              config: {
+                unencrypted: `This value shouldn't get encrypted`,
+              },
+              secrets: {
+                encrypted: 'This value should be encrypted',
+              },
+            })
+            .expect(200)
+        );
+      }
+      const createdActions = await Promise.all(actionPromises);
+      createdActions.forEach((createdAction) =>
+        objectRemover.add(Spaces.space1.id, createdAction.body.id, 'action', 'actions')
+      );
+
+      const pattern = {
+        instance: [true, true, true, false, true, true],
+      };
+      const alertData = getTestAlertData({
+        rule_type_id: 'test.patternFiring',
+        params: { pattern },
+        schedule: { interval: '1m' },
+        throttle: null,
+        notify_when: 'onActiveAlert',
+        actions: createdActions.map((createdAction) => {
+          return {
+            id: createdAction.body.id,
+            group: 'default',
+            params: {
+              index: ES_TEST_INDEX_NAME,
+              reference: '',
+              message: 'test message',
+            },
+          };
+        }),
+      });
+      const { body: createdAlert } = await supertest
+        .post(`${getUrlPrefix(Spaces.space1.id)}/api/alerting/rule`)
+        .set('kbn-xsrf', 'foo')
+        .send(alertData)
+        .expect(200);
+      objectRemover.add(Spaces.space1.id, createdAlert.id, 'rule', 'alerting');
+
+      const events = flatten(
+        await Promise.all(
+          createdActions.map(async (createdAction) => {
+            return await retry.try(async () => {
+              return await getEventLog({
+                getService,
+                spaceId: Spaces.space1.id,
+                type: 'action',
+                id: createdAction.body.id,
+                provider: 'actions',
+                actions: new Map([['execute', { gte: 1 }]]),
+              });
+            });
+          })
+        )
+      );
+
+      const executeActionsEvents = getEventsByAction(events, 'execute');
```

Comment:
Too bad we can't tell if the tasks were executed ephemerally or not.  Either via a top-level mustache variable, or via some property in the event log.  Neither of those make much sense to me _right now_, but I'd bet we'll find a need/desire for something like that, at some point, so maybe we'll have a reason that makes sense later.

Is there some way we can make use of task manager health here?  The problem is that it's going to be poluted with results from the previous runs.  Or maybe a server side endpoint that returns usage data, we call it before and after, make sure some tasks were run ephemerally.  

I don't think this is a blocker, and we shouldn't spend a lot of time on it; opening an issue to enhance this later WORKSFORME.